### ISSUE TITLE
Support all image mime types for detecting and adding alt text in admin

### ISSUE THREAD

**Issue by knutsp** [joedolson/wp-accessibility#45]
**Describe the bug**
In admin Media list view, column Accessibility should show missing alt text for all supported image mime types.

**To Reproduce**
Steps to reproduce the behaviuor:
1. Go to 'Media'
2. Click on 'Add Media file'
3. Upload a file containg a webp image, like 'my.webp'.
4. Save, without adding any alt text
5. Select the list view
6. Accessibility for the newly iploaded attachment shows 'N/A'.

**Expected behavior**
It shoild show "Missing alt" and "Add alt text" button

Same for 'image/avif' and any type of image supported or registered
